### ISSUE TITLE
Memory allocation error

### ISSUE THREAD

**Issue by akshu281** [taohan10200/IIM#32]
Hello @taohan10200, I tried to use this framework on the custom dataset and I get the below error in trainer.py

 crop_imgs, crop_gt, crop_masks = map(lambda x: torch.cat(x, dim=0), (crop_imgs, crop_gt, crop_masks))

![image](https://user-images.githubusercontent.com/35450160/235575109-fdccbef7-7c83-411b-adaa-6ddbdd5bdce4.png)

Any workaround to solve this? I use a 40GB RAM to do this training

**Comment by aviendre** [comment]
Hi, how did you manage to solve this problem as I am running on 59 GB RAM
